src/library/Uno.Material/Styles/Controls/v2/DatePicker.xaml

```diff
@@ -214,53 +292,54 @@
 					<Grid x:Name="LayoutRoot">
 						<!--  Flyout Button  -->
 						<Button x:Name="FlyoutButton"
```

Comment:
We should be changing this to match more of how the Fluent theme is setting the brushes. Looking [here](https://github.com/unoplatform/uno/blob/1ecff68b4c71dfd1f0f7df60fbbed9dd27ac1952/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/DatePicker_themeresources.xaml#L27-L41) we have `DatePickerButtonForeground`, `DatePickerButtonBackground`, and `DatePickerButtonBorderBrush` that should be set as the Foreground/Background/BorderBrush properties on the DatePicker style and should then be template-bound to the inner FlyoutButton of the Template like you see [here](https://github.com/unoplatform/uno/blob/1ecff68b4c71dfd1f0f7df60fbbed9dd27ac1952/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/DatePicker_themeresources.xaml#L330-L332)

And so we should match our MaterialDatePickerFlyoutButtonStyle to the[ Fluent DatePickerFlyoutButtonStyle](https://github.com/unoplatform/uno/blob/1ecff68b4c71dfd1f0f7df60fbbed9dd27ac1952/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/DatePicker_themeresources.xaml#L184-L275) so that it is using the same brush keys in all of the visual states. Notice how the Foreground/Background/BorderBrush resources are set and template-bound from the DatePickerStyle

Reply:
fixed